.eslintrc.cjs

```diff
@@ -1,17 +1,10 @@
 // eslint-disable-next-line @typescript-eslint/no-var-requires
-const config = require('./.prettierrc');
+const config = require('./.prettierrc.cjs');
```

Comment:
We switched to ESM for our application. To opt out of this, we can mark prettier and ESLint as `cjs` (common JS) files as they don't seem to support ESM in our current set up.